packages/lesswrong/server/resolvers/tagResolvers.ts

```diff
@@ -189,6 +191,31 @@ augmentFieldsDict(Tags, {
       }
     }
   },
+  descriptionHtmlWithAnchors: {
+    resolveAs: {
+      type: "String",
+      resolver: async (tag: DbTag, args: {}, context: ResolverContext): Promise<string> => {
+        const description = tag?.description?.originalContents?.data;
+        if (!description) {
+          return "";
+        }
+
+        const cache: Record<string, number> = {};
+        const $ = cheerio.load(description);
+        $(".table strong").each((i, elem) => {
+          const target = $(elem);
+          let slug = slugify(target.html() ?? "");
+          if (cache[slug]) {
+            slug += `-${++cache[slug]}`;
+          } else {
+            cache[slug] = 1;
+          }
+          target.attr("id", slug);
+        });
+        return $.html();
```

Comment:
Why use this code instead of `extractTableOfContents` which also adds the headers? That way the links will be formatted the exact same way they will be as if/when we add a proper ToC to the page?

Reply:
1 sounds good to me.